src/cpp/src/block_manager.hpp

```diff
@@ -6,20 +6,25 @@
 #include <memory>
 #include <list>
 #include <map>
+#include <time.h>
 
 #include "sequence_group.hpp"
 
 namespace ov::genai {
 class KVCacheBlock {
     int m_ref_count;
     int m_index;
+    size_t m_hash;
+    size_t m_num_hashed_tokens;
+    time_t m_timestamp;
```

Comment:
Why not `std::chrono`?

Reply:
No particular reason, I just need to store a timestamp in this field. Do you think it's better to use std::chrono?

Reply:
chrono is better. It has more verbose interface and higher than 1 second precision.

Reply:
Replaced with chrono.